requirements.in

```diff
@@ -44,7 +44,7 @@ ua-parser==0.8.0
 user-agents==2.0
 user-util==0.1.5
 uwsgi
-wagtail==2.12.5
+wagtail==2.13.4
```

Comment:
I think we should try to take the wagtail to the latest version if possible. If not, we should at least get to the latest for `2.13` which is I think `2.13.5` ATM.